pheps/phep-0001.md

```diff
@@ -0,0 +1,287 @@
+
+Vetting an idea publicly before going as far as writing a PHEP is meant to save the potential author time. Asking the community first if an idea is original helps prevent too much time being spent on something that is guaranteed to be rejected based on prior discussions (searching the Internet does not always do the trick). It also helps to make sure the idea is applicable to the entire community and not just the author. Just because an idea sounds good to the author does not mean it will work for most people in most areas.
+
+Once the champion has asked the PyHC community as to whether an idea has any chance of acceptance, a draft PHEP should be presented to the appropriate venue mentioned above. This gives the author a chance to flesh out the draft PHEP to make it properly formatted, of high quality, and to address initial concerns about the proposal.
+
+## Submitting a PHEP
+The proposal must be submitted as a draft PHEP via a [GitHub pull request](https://github.com/heliophysicsPy/standards/pulls). The draft must be written in PHEP style as described below, else it will fail review immediately (although minor errors may be corrected by the editors).
+
+The standard PHEP workflow is:
+
+* You, the PHEP author, fork the PHEP repository, and create a file named `phep-9999.md` that contains your new PHEP. Use "9999" as your draft PHEP number.
+* In the "Type:" header field, enter "Standards Track", "Informational", or "Process" as appropriate, and for the "Status:" field enter "Draft". For full details, see [PHEP Header Preamble](#phep-header-preamble).
+* Update [.github/CODEOWNERS](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners) such that any co-author(s) or sponsors with write access to the standards repository are listed for your new file. This ensures any future pull requests changing the file will be assigned to them.
+* Push this to a branch (not `main`) of your GitHub fork and submit a draft pull request.
+* The PHEP editors review your PR for structure, formatting, and other errors. Approval criteria are:
+    * It is sound and complete. The ideas must make technical sense. The editors do not consider whether they seem likely to be accepted.
+    * The title accurately describes the content.
+    * The PHEP's language (spelling, grammar, sentence structure, etc.) should be correct and conformant. PHEPs with invalid Markdown (MD) markup will not be approved; here "invalid" include Markdown which provides clearly unintended output (in the discretion of the editors).
```

Comment:
We could add pre-commit hooks that check spelling (i.e., codespell) as well as Markdown syntax to automate a lot of this.

Reply:
Yeah, I'd be up for all sorts of machinery here. I don't think the PHEP should _mandate_ it but the more automated, the better.

Reply:
I could cook up all sorts of GPT-4 powered machinery for checking stuff like grammar, spelling, etc...

Reply:
Something to discuss in #24